src/commands/install.rs

```diff
@@ -37,10 +35,8 @@ pub fn install() -> Result<()> {
         Err(_) => bail!("Failed to fetch latest fuel-core release tag from GitHub API"),
     };
 
-    let fuelup_path = home_dir().unwrap().join(Path::new(FUELUP_BIN_PATH));
-
-    if !fuelup_path.is_dir() {
-        fs::create_dir_all(&fuelup_path)?;
+    if !fuelup_bin_dir().is_dir() {
```

Comment:
```suggestion
    let fuelup_bin_dir = fuelup_bin_dir();
    if !fuelup_bin_dir.is_dir() {
```
Constructing `PathBuf` performs dynamic allocation (much like Vec and String). It doesn't matter much in this case but as general good practise it would be better to construct it once and re-use it when it's trivial to do so.